src/CookieTrait.php

```
@@ -295,59 +293,22 @@ protected function cookieGetAll(): array {
-        $cookie = [];
-        [$name, $value] = explode('=', $part, 2);
-        $cookie['name'] = $name;
-        $cookie['value'] = rawurldecode($value);
-        $cookie['secure'] = FALSE;
-        $cookies[] = $cookie;
-      }
+    else {
+      // @codeCoverageIgnoreStart
+      throw new \Exception('Unsupported driver for cookie retrieval.');
+      // @codeCoverageIgnoreEnd
     }
```

PHPMD (2.15.0)

The method cookieGetAll uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)